example/rbac/create_role.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
`/usr/bin/env bash` .

Can you also provide docs what this script does and how to use it?